sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableIO.java

```diff
@@ -462,7 +493,7 @@ public Write withBigtableOptions(BigtableOptions.Builder optionsBuilder) {
      */
     public Write withTableId(String tableId) {
       checkNotNull(tableId, "tableId");
-      return new Write(options, tableId, bigtableService);
+      return new Write(options, ValueProvider.StaticValueProvider.of(tableId), null, null, bigtableService);
```

Comment:
I believe this can just be `StaticValueProvider.of(tableId)` since you've imported that.